Dockerfile-mysterium-node

```diff
@@ -1,15 +1,21 @@
 FROM ubuntu:latest
 MAINTAINER Valdas Petrulis <petrulis.valdas@gmail.com>
 
-# e.g. PACKAGE_FILE="build/package/mysterium-node_0.0.1_linux_amd64.deb"
-ARG PACKAGE_FILE
-COPY $PACKAGE_FILE /tmp/mysterium-node.deb
+ENV VERSION 0.0.6
+ENV NODE 321
```

Comment:
Not good hardcoding NODE key, it should be injected with container environment variables